src/module.c

```diff
@@ -46,18 +49,50 @@ static int _Setup_ThreadPOOL(int threadCount) {
 	return 1;
 }
 
+// Sets the golbal variable of the redis major version
```

Comment:
Typo on "global", but this comment should be much more comprehensive in general.